core/shared/src/main/scala/org/typelevel/log4cats/extras/ConsoleLogger.scala

```diff
@@ -0,0 +1,323 @@
+/*
+ * Copyright 2018 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.log4cats.extras
```

Comment:
We already have the dedicated `js-console` module that exposes `ConsoleLogger` for JS. Shouldn't we either create a new module for JVM/Native to maintain consistency (perhaps `extras` would be fine), or at the very least move the new `ConsoleLogger` to a JVM/Native-specific submodule, so it's only published for the mentioned platforms?

Reply:
I don't really have the setup, but I'm kind of curious what this would do in JS. I'd assume that `Console` has an implementation on that platform 🤔 

It's all going on stderr, so I'd be open to renaming it `StdErrLogger` to avoid confusion.

Reply:
I thought part of the goal was to have this available in core, without depending on other modules?  Otherwise, I'd think we'd just use slf4j and slf4j-simple?

Reply:
That is a core motivation - make it dead simple to use out-of-the-box, and avoid requiring users who don't need heavy configuration (like scastie examples or scripts) to remember the artifact coordinates for the stuff to make `slf4j-simple` work.